CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format is based on [Common Changelog](https://common-changelog.org).\
+Unlike Common Changelog, the `unreleased` section of [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) is preserved with the addition of a tag to specify which type of release should be published and to foster discussions about it inside pull requests. This tag should be one of the following: `[patch]`, `[minor]`, `[major]`. Ex `Unreleased [minor]`.
+
+## Unreleased [minor]
+
+### Added
+
+- Automate release with GitHub Actions
```

Comment:
This is of little interest to reusers. I believe it would be more useful to document what impacts reusers 🙂 
```suggestion
- Publish package on NPM under name `opentermsarchive`.
- Export `filter`, `pageDeclaration` and `fetch` in NPM module.
- Add changelog.
```